src/stores/widgets/StopGapWidget.ts

```diff
@@ -471,6 +492,14 @@ export class StopGapWidget extends EventEmitter {
         this.feedEvent(ev);
     };
 
+    private onStateUpdate = (ev: MatrixEvent): void => {
+        if (this.messaging === null) return;
+        const raw = ev.getEffectiveEvent();
+        this.messaging.feedStateUpdate(raw as IRoomEvent).catch((e) => {
```

Comment:
Just to double check this:
the actual permission check happens in `feedStateUpdate`?
Up to this point we only have checked for the correct room?

Reply:
Correct, matrix-widget-api's `feed` APIs handle all permission checks for us as [a comment above](https://github.com/element-hq/element-web/blob/de3d553ce4779dceb36fcfeaf7fbfd1545098d00/src/stores/widgets/StopGapWidget.ts#L346) reminds